### ISSUE TITLE
Monterey

### ISSUE THREAD

**Issue by xjwan110** [itsluquis/ASUS-S14-X411-OpenCore#1]
Does your computer sleep wake up normally on Monterey？

**Comment by notluquis** [comment]
No, currently CpuTscSync kext is broken on Monterey, so after sleep the kext doesn't work.
